src/examples/order/cancel.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Mercado Pago Process Order.
+ *
+ * @see {@link [TODO: insert Order documentation URL] Documentation }.
+  */
+
+import { Order } from '@src/clients/order';
+import MercadoPago from '@src/index';
+
+const mercadoPagoConfig = new MercadoPago({ accessToken: '<ACCESS_TOKEN>', options: { timeout: 5000 } });
+
+const order = new Order(mercadoPagoConfig);
+
+/**
+ * Creates an order and returns its ID.
+ * @returns {Promise<string>} 
+ */
+async function createOrder(): Promise<string> {
+	try {
+		const orderResponse = await order.create({
+			body: {
+				type: 'online',
+				processing_mode: 'automatic',
+				total_amount: '100.00',
+				external_reference: 'ext_ref_1234',
+				type_config: {
+					capture_mode: 'manual'
+				},
+				transactions: {
+					payments: [
+						{
+							amount: '100.00',
+							payment_method: {
+								id: 'master',
+								type: 'credit_card',
+								token: '<CARD_TOKEN>',
+								installments: 1
+							}
+						}
+					]
+				}
```

Comment:
O exemplo não está funcionando pois faltou o objeto payer com o email na hora de criar a Order.
Eu vi que o exemplo de capture também está sem o payer, consegue arrumar ele também por favor?
```suggestion
				},
				payer: {
					email: 'test_1731350184@testuser.com'
				}
```